docs/endaq/quickstart.rst

```diff
@@ -37,7 +37,7 @@ Most common properties are read-only attributes of :py:class:`endaq.device.Recor
    '2.0'
 
 Some :py:class:`endaq.device.Recorder` properties are identical to those of an
-`idelib.Dataset <https://mide-technology-idelib.readthedocs-hosted.com/en/latest/idelib/dataset.html#idelib.dataset.Dataset>`_
+`idelib.Dataset <https://mide-technology-idelib.readthedocs-hosted.com/en/feature-update-docs/idelib/dataset.html#idelib.dataset.Dataset>`_
```

Comment:
This probably needs a fix in the `idelib` Sphinx setup, but this makes the link work for now.